res/controllers/Roland_DJ-202-scripts.js

```diff
@@ -246,11 +246,7 @@ DJ202.Deck = function (deckNumbers, offset) {
     this.hotcueButton = [];
     this.samplerButton = [];
     this.loopButton = [];
-    
-    samplerNumber = [];
-    samplerNumber[1] = [1,2,3,4, 9,10,11,12]
```

Comment:
When to 2 sampler rows are enabled to show all 16 samplers, I'd like their possition to match the buttons. That's why I mapped it like this. It doesn't matter though if we go to 8 samplers as proposed in #14